platform/master/R/master.R

```diff
@@ -217,6 +217,11 @@ distributedR_start <- function(inst=0, mem=0,
   if (!is.null(pm)){
     stop("distributedR is already running. Call distributedR_shutdown() to terminate existing session\n")
   }
+
+  if(cluster_conf=="" && Sys.getenv(c("DR_CLUSTER_CONF")) != ""){
+    cluster_conf <- Sys.getenv(c("DR_CLUSTER_CONF"))
+  }
+
```

Comment:
Can you combine this code with the code where cluster config file is assigned from DISTRIBUTEDR_HOME, so that cluster_conf is assigned to correct value in one unified place, rather than different places in the function.